.github/workflows/release.yml

```diff
@@ -17,39 +17,36 @@
 # under the License.
 #
 
-name: Post commit - Publish Pulsar Helm Chart
+name: Release Pulsar Helm Charts
 on:
+  workflow_dispatch:
   push:
     branches:
-      - master 
+      - master
+
 jobs:
   release:
     runs-on: ubuntu-latest
     timeout-minutes: 45
+    permissions:
+      contents: write
     steps:
-      - name: checkout
-        uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v3
         with:
-          token: ${{ secrets.PULSARBOT_TOKEN }}
           fetch-depth: 0
 
-      - name: Tune Runner VM
-        uses: ./.github/actions/tune-runner-vm
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
-      - name: Detect changed files
-        id:   changes
-        uses: apache/pulsar-test-infra/paths-filter@master
+      - name: Install Helm
+        uses: azure/setup-helm@v1
```

Comment:
This action is too complex to bundle. An INFRA ticket already adds this action to the whitelist: https://issues.apache.org/jira/browse/INFRA-23716